packages/ui/src/components/editor/panels/Files/container/index.tsx

```diff
@@ -306,11 +311,6 @@ const FileBrowserContentPanel: React.FC<FileBrowserContentPanelProps> = (props)
       })
 
       ClickPlacementState.setSelectedAsset(params.url)
-    } else {
-      if (event.detail === 2) {
```

Comment:
Since this isn't very clear what functionality we are using. Do you mind commenting what event.detail 2 is?

Reply:
event.detail === 2 was handling double-clicks